app/models/person.rb

```diff
@@ -6,7 +6,7 @@ class Person < ApplicationRecord
 
   validate :personable_id_unchanged, on: :update
   validates_associated :personable
-  validates :email, presence: true, uniqueness: true
+  validates :email, email: true, presence: true, uniqueness: true
```

Comment:
I did a double take on this line and was pretty sure it was a mistake, then after asking chatgpt I realized it's referencing the custom validator. Can we put that at the end and rename the custom validator? So:

validates :email, presence: true, uniqueness: true, valid_email: true

(and then rename the validator to ValidEmailValidator, a bit wordy but I think it makes the validates line much clearer)

Reply:
Agreed, let's just remove the custom validator and go with the `format: { with: `